core/src/main/java/com/orange/ouds/core/theme/OudsSizes.kt

```diff
@@ -368,4 +379,5 @@ val OudsSizeKeyToken.value: Dp
         is OudsSizeKeyToken.Icon.WithBody -> OudsTheme.sizes.fromToken(this)
         is OudsSizeKeyToken.Icon.WithLabel -> OudsTheme.sizes.fromToken(this)
         is OudsSizeKeyToken.MaxWidth -> OudsTheme.sizes.fromToken(this)
+        is OudsSizeKeyToken.MinInteractiveArea -> OudsTheme.sizes.fromToken(this)
```

Comment:
If I'm correct this is the first time we encounter this case, where a single value is the sibling of multiple values contained in a data class, and I think we can directly return `OudsTheme.sizes.minInteractiveArea` and remove `OudsSizes.fromToken(token: OudsSizeKeyToken.MinInteractiveArea)`because it always return the same value.